src/comment.rs

```diff
@@ -51,6 +63,8 @@ pub fn rewrite_comment(orig: &str,
             ("/// ", "", "/// ")
         } else if orig.starts_with("//!") || orig.starts_with("/*!") {
             ("//! ", "", "//! ")
+        } else if is_custom_comment(orig) {
+            (&orig[0..4], "", &orig[0..4])
```

Comment:
This seems like it would go wrong if the forth char is not a space, and that is not checked in `is_custom_comment`